README.md

```diff
@@ -12,21 +12,28 @@ Use `python -m http.server` (`python3` if `python` is an alias for python v2) wi
 
 # Widget Status
 
-Below is a table which outlines the general functionality of each widget on each platform. :heavy_check_mark: means that the widget works as expected, :white_check_mark: means that only some functionality works, :x: means that the widget basically does not work at all, and :question: means that the status has not been verified. See the sections dedicated to each widget below for more information about the widgets labelled with :white_check_mark: and :x:.
-
-| Widget | JupyterLab=2 | JupyterLab=3 | Thebe | CKEditor Binger Plugin | libretexts.org |
-|-|-|-|-|-|-|
-| Bokeh | :heavy_check_mark: | :question: | :x: | :x: | :x: |
-| ipywidgets | :heavy_check_mark: | :question: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
-| bqplot | :heavy_check_mark: | :question: | Waiting on Thebe PR #330 | Waiting on Thebe PR #330 | Waiting on Thebe PR #330 |
-| ipyleaflet | :heavy_check_mark: | :question: | :heavy_check_mark: | :white_check_mark: | :white_check_mark: |
-| pythreejs | :heavy_check_mark: | :question: | :heavy_check_mark: | :white_check_mark: | :white_check_mark: |
-| ipympl | :heavy_check_mark: | :question: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| plotly | :heavy_check_mark: | :question: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| ipycytoscape | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| ipyvolume | :heavy_check_mark: |:question: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| ipygany | :heavy_check_mark: | :question: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| nglview | :heavy_check_mark: | :question: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+Below is a table which outlines the general functionality of each widget on each platform. Where applicable, the latest version of the package environment to be verified for that platform has been specified.
+
+  - :heavy_check_mark: means that the widget works exactly as expected 
+  - :o: means that the widget itself is fine, but the platform makes it difficult to use
+  - :x: means that the widget basically does not work at all, not displaying or something of that sort 
+  - :question: means that the status has not been verified. 
+
+See the sections dedicated to each widget below for more information about the widgets labelled with :o: and :x:.
+
+| Widget | JupyterLab=2 | JupyterLab=3 | Thebe | CKEditor Binger Plugin | query.libretexts.org | libretexts.org libraries |
```

Comment:
Can you make a list above that just defines what each of these mean? For example:


- JupyterLab=2: Runs on jupyter.libretexts.org jupyterhub (v=2) using packages install with our default env
- ...

Just so we all know precisely what setup was checked.

Thanks!